lib/ansible/modules/remote_management/manageiq/manageiq_provider.py

```diff
@@ -181,6 +181,19 @@
         required: false
         default: null
         description: The CA bundle string with custom certificates. defaults to None.
+
+  auth_type:
+    required: false
+    description: Auth type endpoint configuration information. E.g. RSA key pair
+    default: null
+    version_added: "2.5"
+    suboptions:
+      userid:
+        description: User ID for auth type.
+        required: true
+      auth_key:
+        description: Key for auth type.
```

Comment:
Again, what does "for auth type" mean here?  What kind of key?  Is this always an SSH key?  Public or private?

Reply:
Yes, private key for connecting to hosts. Have updated description.